changelogs/unreleased/gh-4675-move-info-messages-to-info-logging-level.md

```diff
@@ -0,0 +1,4 @@
+## feature/lua/log
```

Comment:
There's no such section. Please use `feature/core`, because the patch has nothing to do with Lua. In general, we don't use nested sections. For a list of standard sections, please check out the gen-release-notes script:

https://github.com/tarantool/tarantool/blob/b99ac0b4abd2b1beaf7e14123107fd07fb8db4ec/tools/gen-release-notes#L15-L17

(we probably need to add an automated check for this)

Also, according to our [guideline](https://www.tarantool.io/en/doc/latest/contributing/release_notes/), changelog messages should be written in Past Simple. Let's rephrase:

```md
## feature/core

* Changed log level of some information messages from critical to info
  (gh-4675).
```

Also, please don't set the 'full-ci' label for this patch - it's clear that it doesn't break anything - no need to put extra load on CI.

Reply:
Thank you for the detailed response. Fixed